pages/contact.html

```diff
@@ -6,24 +6,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="./../css/contact.css" />
     <link rel="stylesheet" href="./../css/style.css" />
-
+    <link rel="stylesheet" type="text/css" href="../css/home.css">
     <link
       href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap"
       rel="stylesheet"
     />
     <title>Contact - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
+    <nav id="main-nav" class="main-nav">
+      <div class="container main-nav__container">
+        <div id="main-nav__logo" class="main-nav__logo">
+          <img class="main-nav__logo__image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
```

Comment:
i think the class name is incorrect, a better approach could be `main-nav__logo-image`